package.json

```diff
@@ -36,7 +36,7 @@
     "test:contracts:watch": "npm run start:blockchain:client & truffle migrate --reset --compile-all && truffle watch --network development",
     "test:contracts:e2e": "npm run start:blockchain:client & truffle migrate --reset --compile-all && truffle test test-system/end-to-end.js --network development",
     "test:contracts:smoke": "truffle compile && truffle test ./test-smoke/colony-storage-consistent.js --network storageSmoke",
-    "test:security:slither": "slither . --solc-disable-warnings --exclude-low --exclude-informational --exclude-optimization --exclude uninitialized-state-variables,uninitialized-local-variables --filter-paths 'lib|contracts/testHelpers'",
+    "test:security:slither": "slither . --solc-disable-warnings --exclude-low --exclude-informational --exclude-optimization --exclude uninitialized-state-variables,uninitialized-local-variables,unused-return --filter-paths 'lib|contracts/testHelpers'",
```

Comment:
I'm not wild about doing this, this is a legitimate mistake which it's usefully flagged before. Looking at the output from a failed build, maybe disabling the rule for the function in question would work?

Reply:
The issue is that there are two exclusions necessary for that line -- something about the try/catch logic throws up `unused return` and `unchecked transfer`. I've tried every way I can think of (two consecutive lines of ignore, including two detectors in one exclude) but slither doesn't seem to be able to handle it.

Reply:
Didn't I get rid of this at some point and it still worked?

Reply:
Mm, I think I remember that. Let's give it a go.